airbyte-server/src/test/java/io/airbyte/server/handlers/ConnectionsHandlerTest.java

```diff
@@ -312,52 +327,270 @@ void testCreateConnectionWithBadDefinitionIds() throws JsonValidationException,
     }
 
     @Test
-    void testUpdateConnection() throws JsonValidationException, ConfigNotFoundException, IOException {
-      final AirbyteCatalog catalog = ConnectionHelpers.generateBasicApiCatalog();
-      catalog.getStreams().get(0).getStream().setName("azkaban_users");
-      catalog.getStreams().get(0).getConfig().setAliasName("azkaban_users");
+    void testUpdateConnectionPatchSingleField() throws Exception {
```

Comment:
it's probably worth throwing all the update connection patch related test into an inner test class so it's clearer these share a group category. See the `StreamConfigurationDiff` class in this class.